samples/contrib/aws-samples/simple_train_pipeline/training-pipeline.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import kfp
+from kfp import components
+from kfp import dsl
+from kfp.aws import use_aws_secret
+
+sagemaker_train_op = components.load_component_from_file('../../../../components/aws/sagemaker/train/component.yaml')
+
+@dsl.pipeline(
+    name='Training pipeline',
+    description='SageMaker training job test'
+)
+def training(
+        region='us-east-1',
+        endpoint_url='',
+        image='382416733822.dkr.ecr.us-east-1.amazonaws.com/kmeans:1',
+        training_input_mode='File',
+        hyperparameters='{"k": "10", "feature_dim": "784"}',
+        channels='[                                                               \
+                    {                                                             \
+                      "ChannelName": "train",                                     \
+                      "DataSource": {                                             \
+                        "S3DataSource": {                                         \
+                          "S3Uri": "s3://m-kfp-mnist/mnist_kmeans_example/data",  \
```

Comment:
The bucket name `m-kfp-mnist` is hardcoded and it's not a public bucket.  Please parametrize it as in other parts of this example.

Reply:
that's just the default value (not hardcoded)

changed it to "s3://kubeflow-pipeline-data/mnist_kmeans_example/data" to match [other examples](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py#L32)